video/out/vo.c

```diff
@@ -890,7 +890,7 @@ static bool render_frame(struct vo *vo)
     in->dropped_frame &= frame->can_drop;
     // Even if we're hopelessly behind, rather degrade to 10 FPS playback,
     // instead of just freezing the display forever.
-    in->dropped_frame &= now - in->prev_vsync < 100 * 1e6;
+    in->dropped_frame &= now - in->prev_vsync < MP_TIME_S_TO_NS(0.1);
```

Comment:
```suggestion
    in->dropped_frame &= now - in->prev_vsync < MP_TIME_MS_TO_NS(100);
```
to avoid double conversion, not needed.